libraries/rush-lib/src/api/test/__snapshots__/RushCommandLine.test.ts.snap

```diff
@@ -176,6 +176,14 @@ Object {
           "required": false,
           "shortName": undefined,
         },
+        Object {
+          "description": "(EXPERIMENTAL) Specifies a Rush subspace to be installed. Requires the \\"subspacesEnabled\\" feature to be enabled in subspaces.json.",
```

Comment:
This documentation is incorrect. It should say something like:

`(EXPERIMENTAL) Specifies an individual Rush subspace to check, requiring versions to be consistent only within that subspace (ignoring other subspaces). If this parameter is omitted, then \"rush check\" will instead require consistency across all projects in the monorepo. Requires the \"subspacesEnabled\" setting to be true in subspaces.json.`

Reply:
Updated the documentation for rush check and rush version to this